src/FlaUI.WebDriver.UITests/SessionTests.cs

```diff
@@ -315,6 +315,35 @@ public void NewCommandTimeout_NotExpired_DoesNotEndSession()
             Assert.That(() => driver.Title, Throws.Nothing);
         }
 
+        [Test, Explicit(("Sometimes multiple processes are left open"))]
+        public void NewCommandTimeout_Expired_DoesNot_CloseApp_AppTopLevelWindowTitleMatch()
```

Comment:
The test method naming convention I used so far is with exactly two underscores: `MethodUnderTest_Condition_ExpectedOutcome`. So in this case I would suggest
`NewCommandTimeout_SessionWithAppTopLevelWindowTitleMatch_ClosesSessionButDoesNotCloseApp`

Reply:
Done! Thanks, wasn't quite sure what the convention was.